nilearn/image/image.py

```diff
@@ -638,7 +638,9 @@ def new_img_like(ref_niimg, data, affine=None, copy_header=False):
         header = copy.deepcopy(ref_niimg.header)
         header['scl_slope'] = 0.
         header['scl_inter'] = 0.
-        header['glmax'] = 0.
+        # "glmax" is available only for Nifti1Image
+        if not isinstance(ref_niimg, nibabel.nifti2.Nifti2Image):
```

Comment:
Wouldn't it simply work to do "if 'glmax' in header"?

Reply:
That's good point. I am changing that.

Reply:
Thanks